src/content/developers/docs/networking-layer/portal-network/index.md

```diff
@@ -0,0 +1,93 @@
+
+Light client's can't currently request specific pieces of chain data over DevP2P or libP2p because those protocols are only designed to enable chain synchronization and gossiping of blocks and transactions. Light clients do not want to download this information because that would stop them from being "light".
+
+The JSON-RPC API is not an ideal choice for light client data requests either, because it relies upon a connection to a specific full node or centralized RPC provider that can serve the data. This means the light client has to trust that specific node/provider to be honest, and also the full node might have to handle lots of requests from many light clients, adding to their bandwidth requirements.
+
+Portal decided to rethink the whole design.
+
+“What if we could build outside of these design constraints.”
+
+The core idea of the Portal Network is to take the best bits of the current networking stack by enabling information needed by light clients, such as historical data and the identity of the current head of the chain to be served through a lightweight DevP2P style peer-to-peer decentralized network.
+
+The idea is to add small parts of the total historical Ethereum data and some specific node responsibilities to each node. Then, requests are served by seeking out the nodes storing the specific data that was requested and retrieving it from them. 
+
+This inverts the normal model of light nodes finding a single node and requesting them to filter and serve large volumes of data; instead, they quickly filter a large network of nodes that each handle small amounts of data.
+
+The goal is to allow a decentralized network of lightweight Portal clients to:
+
+- track the head of the chain
+- sync recent and historical chain data
+- retrieve state data
+- broadcast transactions
+- execute transactions using the [EVM](https://ethereum.org/en/developers/docs/evm/)
+
+
+The benefits of this network design are:
+
+- Reduce Internet bandwidth usage
+- Minimized or zero syncing
+- Accessible to resource-constrained devices (<1GB ram, <100mB disk, 1CPU)
+- Residential internet connection
+
+The diagram below shows the functions of existing clients that can be delivered by the Portal Network, enabling users to access these functions on very low-resource devices.
+
+![portal network table](portal-network-table.png)
+
+
+
+## Client diversity by default {#client-diversity-as-default}
+
+The Portal Network developers also made the design choice to build three separate Portal Network clients from day one.
+
+Having multiple independent client implementations enhances the resilience and decentralization of the Ethereum network.
+
+If one client experiences issues or vulnerabilities, other clients can continue to operate smoothly, preventing a single point of failure. Additionally, diverse client implementations foster innovation and competition, driving improvements and reducing monoculture risk within the ecosystem.
```

Comment:
Is it worth linking to the github repo for each client here?

Reply:
yes, sure, coming up...